x-pack/plugins/monitoring/public/lib/ajax_error_handler.tsx

```diff
@@ -11,7 +11,7 @@ import { EuiButton, EuiSpacer, EuiText } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
```

Comment:
I'm not sure this file is used anymore 😱 
I think it's a leftover from the migration... I'm going to check it now and create a PR to remove it if it's not used.

Reply:
Thanks!

Reply:
Done https://github.com/elastic/kibana/pull/121104